### PR TITLE
Docs Updates

### PR DESCRIPTION
**Pull request by grant**
- Unignores the docs
- Adds `events` and `docker` docs

### CODE REVIEW

**Comment by grant** [comment]
@googlebot I consent.

**Comment by grant** [comment]
@googlebot I consent.

File: docs/events.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+After setup, all notifications to the subscription topic will be pushed to your cloud function.
+
+Here is a sample script for creating subscription with a `pushEndpoint`:
+
+ ```js
+{ PubSub } require('@google-cloud/pubsub');
+
+async function main() {
+  const pubsub = new PubSub({
+    apiEndpoint: 'localhost:8085', // Pubsub emulator endpoint
```
Comment (ace-n) [inline docs/events.md]: I assume `8085` is constant?

File: docs/events.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  http://localhost:8080
+```
+ 
+### Using the Pub/Sub emulator
+
+Another way to test your cloud function Pub/Sub endpoint is to use the [Pub/Sub Emulator](https://cloud.google.com/pubsub/docs/emulator). This allows you to use the Pub/Sub notification from another service to trigger your cloud function.
+
+The high level approach is to:
+1. Start the Pub/Sub Emulator
+2. Use the Pub/Sub client library to create a subscription and set the `pushEndpoint` to `http://localhost:8080`.
```
Comment (ace-n) [inline docs/events.md]: I assume `8080` is constant?

File: docs/events.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+After setup, all notifications to the subscription topic will be pushed to your cloud function.
+
+Here is a sample script for creating subscription with a `pushEndpoint`:
+
+ ```js
+{ PubSub } require('@google-cloud/pubsub');
```
Comment (ace-n) [inline docs/events.md]: missing assignment statement? e.g.
`const { PubSub } = ...`

(@grant - mind testing this code, or at least linting it?)

Comment (grant) [reply docs/events.md]: Fixed this basic mistake.

File: docs/events.md (hunk 1 above)
Comment (grant) [reply docs/events.md]: Yes. It is a constant in the docs linked above.
See:
https://cloud.google.com/pubsub/docs/emulator

File: docs/events.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+## Local Testing of Cloud Events
+
+The setup for cloud functions that accept events is very similar to the instructions in the quickstart, with the following adjustments:
```
Comment (ace-n) [inline docs/events.md]: Nit: link to quickstart?

File: docs/events.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+In your `package.json`, add a signature type (in bold) to your start command:
```
Comment (ace-n) [inline docs/events.md]: Nit: remove? (Maybe italicize _signature type_ instead?)

File: docs/events.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+<pre>
+  "scripts": {
+    "start": "functions-framework --target=helloWorld <b>--signature-type=event"</b>
+  }
+</pre>
+
+Upon running ```sh npm start ```, you'll see the function is still being served at `http://localhost:8080/`. However, it is no longer accessible via GET requests from the browser. Instead, send a POST request where the request body conforms to the API defined by [push subscriptions](https://cloud.google.com/pubsub/docs/push). 
+
+### Submitting POST Request to Simulate Pub/Sub messages
```
Comment (ace-n) [inline docs/events.md]: nit: `Requests` (add s)

File: docs/events.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  -H "Ce-Type: true" \
+  -H "Ce-Specversion: true" \
+  -H "Ce-Source: true" \
+  -H "Ce-Id: true" \
+  http://localhost:8080
+```
+ 
+### Using the Pub/Sub emulator
+
+Another way to test your cloud function Pub/Sub endpoint is to use the [Pub/Sub Emulator](https://cloud.google.com/pubsub/docs/emulator). This allows you to use the Pub/Sub notification from another service to trigger your cloud function.
```
Comment (ace-n) [inline docs/events.md]: Capitalization nit: `cloud function` -> `Cloud Function`?

File: docs/events.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+The high level approach is to:
+1. Start the Pub/Sub Emulator
+2. Use the Pub/Sub client library to create a subscription and set the `pushEndpoint` to `http://localhost:8080`.
+
+After setup, all notifications to the subscription topic will be pushed to your cloud function.
```
Comment (ace-n) [inline docs/events.md]: Nit: would `locally hosted function` be better than `Cloud Function` here?

File: docs/events.md (hunk 4 above)
Comment (grant) [reply docs/events.md]: Done.

File: docs/events.md (hunk 5 above)
Comment (grant) [reply docs/events.md]: Bolded sig type instead as it is important.

File: docs/events.md (hunk 6 above)
Comment (grant) [reply docs/events.md]: Done.

File: docs/events.md (hunk 7 above)
Comment (grant) [reply docs/events.md]: Done.

File: docs/events.md (hunk 8 above)
Comment (grant) [reply docs/events.md]: Done.

**Comment by grant** [comment]
Merging as I have signed the CLA and I am squashing.

**Pull request merged**